### PR TITLE
Update manifest

### PR DESCRIPTION
**Pull request by Kallecrafter**
## Type of change

- [ ] Add new directory for a server
- [x] Update directory of a server
- [ ] Remove directory of a server
- [x] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

**Review by Kallecrafter — COMMENTED** [review]
fixed

File: minecraft_servers/craftergang/manifest.json [hunk 1]
```diff
@@ -1,101 +1,96 @@
...
-        "discord": "https://discord.gg/ThCt8Pq",
-        "teamspeak": "craftergang.de",
-        "tiktok": "craftergang",
-        "youtube": "https://www.youtube.com/@craftergang",
-        "instagram": "craftergang"
+  "server_name": "craftergang",
+  "nice_name": "Craftergang",
+  "direct_ip": "craftergang.de",
+  "server_wildcards": [
+    "%.craftergang.net"
```
Comment (maaaathis) [inline minecraft_servers/craftergang/manifest.json:6]: Doesn't look like a real wildcard:
image

File: minecraft_servers/craftergang/manifest.json [hunk 2]
```diff
@@ -1,101 +1,96 @@
...
+    "mlgrush": {
+      "name": "MLGRush",
+      "color": "#de0b39",
+      "command": "/gametp mlgrush",
+      "versions": "1.8<1.21"
+    }
+  },
+  "discord": {
+    "server_id": 251791624483373060,
+    "rename_to_minecraft_name": true
```
Comment (maaaathis) [inline minecraft_servers/craftergang/manifest.json:83]: Reservered for partners only; change this back to `false` if you're not a LabyMod partner

**Review by maaaathis — CHANGES_REQUESTED** [review]

**Comment by Kallecrafter** [comment]
image

**Comment by Kallecrafter** [comment]
Yes, but if I don't use %. Then the manifest is considered faulty.

**Comment by Kallecrafter** [comment]
> > image
> 
> Yeah, thats the right place. A wildcard need to start with a `%.`

Here ist the Error:
 Invalid wildcard entry. Each entry must start with **%.**. Further information here: https://en.wikipedia.org/wiki/Wildcard_DNS_record (`server_wildcards`)

**Comment by Nils-afk** [comment]
Just replace the normal with %.craftergang.net

**Comment by maaaathis** [comment]
> Just replace craftergang.net with %.craftergang.net

If it's really a wildcard^^ (https://en.wikipedia.org/wiki/Wildcard_DNS_record)

**Comment by Nils-afk** [comment]
> > Just replace craftergang.net with %.craftergang.net
> 
> If it's really a wildcard^^ (https://en.wikipedia.org/wiki/Wildcard_DNS_record)

He only has to add an "A-Record" The name * and the server IP and that's it

**Comment by Kallecrafter** [comment]
> > > Just replace craftergang.net with %.craftergang.net
> > 
> > 
> > If it's really a wildcard^^ (https://en.wikipedia.org/wiki/Wildcard_DNS_record)
> 
> He only has to add an "A-Record" The name * and the server IP and that's it

And that would look like this?

**Comment by Nils-afk** [comment]
Yes

**Comment by Kallecrafter** [comment]
fixed

File: minecraft_servers/craftergang/manifest.json [hunk 3]
```diff
@@ -1,101 +1,97 @@
...
-        "discord": "https://discord.gg/ThCt8Pq",
-        "teamspeak": "craftergang.de",
-        "tiktok": "craftergang",
-        "youtube": "https://www.youtube.com/@craftergang",
-        "instagram": "craftergang"
+  "server_name": "craftergang",
+  "nice_name": "Craftergang",
+  "direct_ip": "craftergang.de",
+  "server_wildcards": [
+    "%.craftergang.net",
```
Comment (maaaathis) [inline minecraft_servers/craftergang/manifest.json:6]: Not a real wildcard
Google Chrome 2024-08-09 14 58 36

File: minecraft_servers/craftergang/manifest.json [hunk 4]
```diff
@@ -1,101 +1,97 @@
...
+    "%.craftergang.de"
```
Comment (maaaathis) [inline minecraft_servers/craftergang/manifest.json:7]: Not a real wildcard
Shottr 2024-08-09 14 58 43

**Review by maaaathis — CHANGES_REQUESTED** [review]

**Comment by Kallecrafter** [comment]
Server is not Offline!!!
image

**Comment by Kallecrafter** [comment]
image
image

**Comment by Kallecrafter** [comment]
Old Merges from the Server:
image
image

craftergang.net was set in "server_wildcards" and worked flawlessly.

**Comment by Kallecrafter** [comment]
image

**Comment by maaaathis** [comment]
> **I have both the ip craftergang.de and craftergang.net If I set the wildcard without a "%" sign in front of the IP, the system tells me there are problems. If I put one in front of it, you tell me a problem. I don't know what to do. For example, I have craftergang.de as Domain and craftergang.net as subdomain which also works.**

See the definition for a wildcard dns entry: https://en.wikipedia.org/wiki/Wildcard_DNS_record

**Comment by Kallecrafter** [comment]
**Please just merge it. It works perfectly. DoJapHD had always merged it before, with the same domain. I don't know what the problem is.**

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**